CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at semgrep@r2c.dev. All
+reported by contacting the project team at semgrep@semgrep.dev. All
```

Comment:
I'm not sure if this is the correct address in this case; should it be semgrep@semgrep.com? Or something else?

Reply:
Err. Neither of these addresses exists! 

I would suggest we make it `support@semgrep.com`

Reply:
Thanks for checking this, @werrett ! I don't know if that address would be the most efficient, but if there's not a dedicated one currently, we could use it as the best option for now, and consider revising later.

Reply:
@armchairlinguist @werrett  support@semgrep.com already exists and is used by customers to open support cases today. That should be the address we should include for all customer facing error messages.

Reply:
I updated this, but I did want to briefly note it's not an error message - it's guiding text that's part of this repository's Code of Conduct file. I think this is good for now, but it's possible we'd want a dedicated address sometime in the future.